idd/Energy+.idd.in

```diff
@@ -16581,58 +16592,50 @@ SurfaceProperty:ExposedFoundationPerimeter,
        \units dimensionless
        \minimum 0.0
        \maximum 1.0
-       \default 1.0
-  A2 , \field Surface Segment 1 Exposed
+  A3 , \field Surface Segment 1 Exposed
        \note Surface Segment N is the perimeter between the Nth and (N+1)th
        \note vertices
        \begin-extensible
        \type choice
-       \default Yes
        \key Yes
        \key No
-  A3 , \field Surface Segment 2 Exposed
+  A4 , \field Surface Segment 2 Exposed
        \type choice
-       \default Yes
        \key Yes
        \key No
-  A4 , \field Surface Segment 3 Exposed
+  A5 , \field Surface Segment 3 Exposed
        \type choice
        \default Yes
        \key Yes
        \key No
-  A5 , \field Surface Segment 4 Exposed
+  A6 , \field Surface Segment 4 Exposed
        \type choice
-       \default Yes
        \key Yes
        \key No
-  A6 , \field Surface Segment 5 Exposed
+  A7 , \field Surface Segment 5 Exposed
        \type choice
        \default Yes
        \key Yes
        \key No
-  A7 , \field Surface Segment 6 Exposed
+  A8 , \field Surface Segment 6 Exposed
        \type choice
        \default Yes
        \key Yes
        \key No
-  A8 , \field Surface Segment 7 Exposed
+  A9 , \field Surface Segment 7 Exposed
        \type choice
-       \default Yes
        \key Yes
        \key No
-  A9 , \field Surface Segment 8 Exposed
+  A10, \field Surface Segment 8 Exposed
        \type choice
-       \default Yes
        \key Yes
        \key No
-  A10, \field Surface Segment 9 Exposed
-       \type choice
-       \default Yes
+  A11, \field Surface Segment 9 Exposed
+       \ty choice
```

Comment:
It is an error from the new LBNL urban PR.  Grrrr.  Can we patch the fix in this branch so we don't have to have yet another thing to review?